tareas/Andres_Sealtiel/Tarea_2.jl

```diff
@@ -0,0 +1,619 @@
+# -*- coding: utf-8 -*-
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .jl
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.11.4
+#   kernelspec:
+#     display_name: Julia 1.6.2
+#     language: julia
+#     name: julia-1.6
+# ---
+
+# <div style="text-align: center"> 
+#
+# # Temas Selectos de Física Computacional I
+# ## Tarea 2
+# #### Ramos Arzate Andres & Pichardo Jiménez, Sealtiel ####
+#
+# ### Ejercicio. 1    
+# </div>
+
+"""
+    iterados(f, x₀, c, n)
+Función que calcula los *'n'* iterados de la función *'f(x,c)'*, evaluada en la condición inicial
+*'x₀'* y con constante *'c'*.
+"""
+function iterados(f, x₀, c, n)
+    iters = zeros(n+1)
+    iters[1] = f(x₀, c)
+    for i in 1:n
```

Comment:
*Sugerencia de código*

```suggestion
    @inbounds for i in 1:n
```
esto puede acelerar su codigo, ya que de esta forma julia no checa si en efecto `iters` tiene un elemento `i` en `iters[i]`, esto igual lo pueden implementar en más funciones de la tarea